modules/connecttest.lua

```diff
@@ -188,8 +188,13 @@ end
 
 module.timers = { }
 
-function RUN_AFTER(func, timeout)
-  xmlReporter.AddMessage(debug.getinfo(1, "n").name, tostring(func), {["Timeout"] = tostring(timeout)})
+function RUN_AFTER(func, timeout, funcName)
+  func_name_str = "noname"
```

Comment:
@LuxoftAKutsan , this strings is in different methods, in future default names can be different.